### ISSUE TITLE
repository indexing

### ISSUE THREAD

**Issue by Krakozaber** [xelkano/redmine_xapian#57]
If isset
projects = [ 'proj_1', 'proj_2' ]
indexing only first

**Comment by picman** [comment]
I can't reproduce this behaviour.
If I set verbose output using -v I can see each processed project in the output:
```
Mon Jun 22 16:15:30 2015: - Indexing repositories for PROJECT_NAME ...
```
Can you post your command with arguments and corresponding output, please?

**Comment by Krakozaber** [comment]
Fixed with #58
